rpt/operationslist_es.properties

```diff
@@ -0,0 +1,7 @@
+#Generated by Eclipse Messages Editor (Eclipse Babel)
+
+Operation.Type = Operation / Tipología

```

Comment:
There's no need to add new keys in the language other than "en". Please revert this change.